GoogleSignIn/Sources/GIDKeychainHandlling/GIDKeychainHandler.m

```diff
@@ -0,0 +1,41 @@
+#import "GoogleSignIn/Sources/GIDKeychainHandlling/GIDKeychainHandler.h"
+
+#ifdef SWIFT_PACKAGE
+@import AppAuth;
+@import GTMAppAuth;
+#else
+#import <AppAuth/AppAuth.h>
+#import <GTMAppAuth/GTMAppAuth.h>
+#endif
+
+// Keychain constants for saving state in the authentication flow.
```

Comment:
I think the constant's name is clear enough -- the comment isn't needed.

Otherwise, s/constants/constant